### PR TITLE
Update slack-start-guide.md - add email change

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Added small change (thanks to @SaraVilla) about changing email accounts on slack!

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Review by SaraVilla — COMMENTED** [review]
maybe let's mention 2 options: 

- to change account emails, you can do yourself from account settings online, following these instructions: https://slack.com/intl/en-gb/help/articles/207262907-Change-your-email-address

- If you want to merge accounts, workspace admin will have to delete one of them so then you can update your email address as required. 

does this sound correct @aleesteele ?

File: book/website/community-handbook/communication-channels/slack-start-guide.md [hunk 1]
```diff
@@ -83,11 +83,14 @@ If a channel has become too noisy, you can additionally:
 
 * While IM-based conversations can feel very immediate, please don’t feel that you have to respond to messages immediately, nor expect others to respond immediately. We’re all busy, really busy!
 
-
 #### Message Editing & Deletion
 
 * You are allowed to edit your messages at any time. That means if you edit a message after someone replied to it, make it clear that you edited something if it changes the meaning of your message. See the example in the GIF below.
 
+#### Account
```
Comment (aleesteele) [inline book/website/community-handbook/communication-channels/slack-start-guide.md:90]:
```suggestion
#### Account Management
```

**Review by Arielle-Bennett — APPROVED** [review]
This all looks good to me @aleesteele ready to merge!

**Comment by bsipocz** [comment]
Thanks @aleesteele!

File: book/website/community-handbook/communication-channels/slack-start-guide.md [hunk 2]
```diff
@@ -83,11 +83,14 @@ If a channel has become too noisy, you can additionally:
...
 * While IM-based conversations can feel very immediate, please don’t feel that you have to respond to messages immediately, nor expect others to respond immediately. We’re all busy, really busy!
 
-
 #### Message Editing & Deletion
 
 * You are allowed to edit your messages at any time. That means if you edit a message after someone replied to it, make it clear that you edited something if it changes the meaning of your message. See the example in the GIF below.
 
+#### Account Management
+
+* To change the email your slack account is attached to, message a member of _The Turing Way_ team, who will have to delete your account first, so you can join with an alternative email.
```
Comment (SaraVilla) [inline book/website/community-handbook/communication-channels/slack-start-guide.md:92]: To change the email your slack account is attach to, just follow these instructions form Slack help: https://slack.com/intl/en-gb/help/articles/207262907-Change-your-email-address. 
If you need to merge two existing accounts, message message a member of _The Turing Way_ team, who will have to delete your account first, so you can join with your preferred email.

**Pull request merged**